release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -1636,6 +1636,42 @@ With this release, availability sets are no longer modified after creation, allo
+
+* Previously, the *Critical Alerts* section of the *Notification Drawer* could not be collapsed. With this release, the section can be collapsed. (link:https://issues.redhat.com/browse/OCPBUGS-55702[OCPBUGS-55702])
+
+* Previously, when viewing the list of installed Operators, an Operator appeared twice in the list if the currently selected project matched an Operator's default namespace while copied CSVs were disabled in {olm-first}. With this release, the Operator appears only once in such cases. (link:https://issues.redhat.com/browse/OCPBUGS-54601[OCPBUGS-54601])
+
+* Previously, the link to OperatorHub in the *Installed Operators* page would trigger a hard reload. With this release, this link no longer triggers a hard reload. (link:https://issues.redhat.com/browse/OCPBUGS-54536[OCPBUGS-54536])
+
+* Previously, selecting *All Projects* from the project picker while on the *Create VolumeSnapshot* page resulted in a page not found error. With this release, the VolumeSnapshot list page is correctly displayed. (link:https://issues.redhat.com/browse/OCPBUGS-53227[OCPBUGS-53227])
+
+* Previously, there was incorrect logic for calculating the pod container count causing it to be inaccurate. With this release, the `Ready` and `Started` status in count logic was added so the correct pod container count is displayed, which is consist with `oc` CLI. (link:https://issues.redhat.com/browse/OCPBUGS-53118[OCPBUGS-53118])
+
+* Previously, the Select above the Node Logs did not close when opened, unless the Select's toggle was clicked again or one of the Select's menu items was clicked. With this release, the Select closes after clicking outside of the menu or by pressing the appropriate key on the keyboard. (link:https://issues.redhat.com/browse/OCPBUGS-52316[OCPBUGS-52316])
+
+* Previously, the shared Timestamp component referenced an undefined property when calculating relative times. As a result, most times displayed in the console were not correctly displaying relative strings such as "Just now" or "Less than a minute ago". With this release, the issue is fixed and relative time strings are correctly rendered in the console. (link:https://issues.redhat.com/browse/OCPBUGS-51202[OCPBUGS-51202])
+
+* Previously, the observe menu only displayed based on the current user and console configuration for monitoring. This caused other items added by observability plugins to be hidden. With this release, the observe menu also displays items from different observability plugins. (link:https://issues.redhat.com/browse/OCPBUGS-50693[OCPBUGS-50693])
+
+* Previously, when logging in to the console for the first time, automatic perspective detection caused the console to ignore the specific URL path that the user clicked to get onto the console, and instead load a different page. With this release, the current path is followed even when automatic perspective switching occurs. (link:https://issues.redhat.com/browse/OCPBUGS-50650[OCPBUGS-50650])
```

Comment:
I wonder if we should update this to just say "with this release the correct path is followed? Since perspectives unified.